docs/Training-Self-Play.md

```diff
@@ -1,21 +1,40 @@
 # Training with Self-Play
 
-ML-Agents provides the functionality to train symmetric, adversarial games with [Self-Play](https://openai.com/blog/competitive-self-play/).
-A symmetric game is one in which opposing agents are *equal* in form and function. In reinforcement learning,
-this means both agents have the same observation and action spaces.
-With self-play, an agent learns in adversarial games by competing against fixed, past versions of itself
-to provide a more stable, stationary learning environment. This is compared
-to competing against its current self in every episode, which is a constantly changing opponent.
+ML-Agents provides the functionality to train both symmetric and asymmetric adversarial games with
+[Self-Play](https://openai.com/blog/competitive-self-play/).
+A symmetric game is one in which opposing agents are equal in form, function snd objective. Examples of symmetric games
```

Comment:
```suggestion
A symmetric game is one in which opposing agents are equal in form, function and objective. Examples of symmetric games
```

Can you run a spell check on the documentation once you are done modifying it to catch those?